src/content/whats-new/2024/12/whats-new-12-03-fossa.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Gain security insights with FOSSA in New Relic vulnerability management dashboard'
+summary: 'Quickly view FOSSA open-source vulnerability scan results alongside other security detections within the New Relic vulnerability management dashboard.'
+releaseDate: '2024-12-03'
+learnMoreLink: 'https://docs.newrelic.com/docs/vulnerability-management/integrations/fossa/'
+---
+
+We're thrilled to introduce the integration between New Relic and FOSSA, offering users enhanced security insights. This integration enables FOSSA users to view open-source vulnerability scan results directly within the New Relic vulnerability management dashboard, alongside other important security detections. This unified view allows for a more holistic approach to managing vulnerabilities across both build-time and runtime environments.
+
+## Why this is important
+Users of New Relic and FOSSA now benefit from a comprehensive view of their vulnerabilities, alongside health and performance metrics, all within the same New Relic dashboard. This capability ensures that new and existing services meet production readiness compliance and security goals, enhancing overall service reliability and security posture.
+
+## Prioritize what to fix first and what can wait
+By integrating FOSSA detections into the New Relic vulnerability management dashboard, users gain a centralized platform to:
+
+- **Rate vulnerabilities:** Utilize data on exploits in the wild and active ransomware threats to assess vulnerability severity.
+- **Prioritize detections:** Determine which risks require immediate attention and which can be deferred, optimizing resource allocation.
+- **Avoid wasted effort:** Prevent unnecessary developer workload by focusing on vulnerabilities that are actively executed in runtime environments.
```

Comment:
Use "run" or "performed" instead of "executed"

Reply:
Updated